### ISSUE TITLE
Fullscreen Transparency

### ISSUE THREAD

**Issue by owenthewizard** [fluix-dev/sway-borders#65]
> ### Please read the following before submitting:
> - Please try and get your feature implemented into Sway upstream first. This can be omittited if it definitely won't make it, such  as more asthetic features like blur.

See swaywm/sway#4040

> ### Please fill out the following:
> - **Description:**
> Please describe in plain English what the enhancement is and what the use case is.

Working fullscreen transparency for transparent windows, e.g. terminals. See the above Sway issue for examples.

**Comment by owenthewizard** [comment]
See @Hummer12007's [patch](https://l.sr.ht/DG1I.patch) from https://github.com/swaywm/sway/issues/4040#issuecomment-488351885

**Comment by fluix-dev** [comment]
I believe fullscreen applications currently have further optimizations as well, such as direct rendering and skipping the compositing step (I don't know the details). I think using the IPC to create a script which makes the window floating, resizes it to the full screen, and removes borders would achieve the effect you like. Otherwise, some other "fake fullscreen" option(s) might be better, especially considering other fullscreen preferences people have, like not sending fullscreen signals to, say, browsers.
